javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $bodyColor = document.querySelector('body');
+
+function lightBulb(event) {
+  if ($circle.className === 'circle') {
+    $circle.className = 'circle light-bulb-on';
```

Comment:
Instead of checking the current class name, make a boolean variable to use as your data model and base your conditional logic on that.